### ISSUE TITLE
RuntimeError: "slow_conv2d_cpu" not implemented for 'Half'

### ISSUE THREAD

**Issue by OliverZhai1515** [Ekko-zn/AIGCDetectBenchmark#9]
https://github.com/Ekko-zn/AIGCDetectBenchmark/blob/088701099e1385ca51ae63af445ea80f39c702c5/data/process.py#L368-L404

It seems that the variable on the CPU can not be used for 'Half' , moving the `img` to the GPU on line 384 and the `model` to the GPU on line 398 and 389. Then it works.

**Comment by Aemilia-xu** [comment]
You are right. Another way is to set DIRE_args.use_fp16 as False. We will fix this problem. Thank you for your feedback.
